rskj-core/src/test/java/co/rsk/core/bc/TransactionPoolImplTest.java

```diff
@@ -98,7 +98,8 @@ protected RepositoryLocator buildRepositoryLocator() {
                 10,
                 100,
                 Mockito.mock(TxQuotaChecker.class),
-                Mockito.mock(GasPriceTracker.class));
+                Mockito.mock(GasPriceTracker.class),
+                null);
```

Comment:
Same idea with an override constructor can be applied here making this change unnecessary.

```
    public TransactionPoolImpl(RskSystemProperties config, RepositoryLocator repositoryLocator, BlockStore blockStore, BlockFactory blockFactory, EthereumListener listener, TransactionExecutorFactory transactionExecutorFactory, SignatureCache signatureCache, int outdatedThreshold, int outdatedTimeout, TxQuotaChecker txQuotaChecker, GasPriceTracker gasPriceTracker) {
        this(config, repositoryLocator, blockStore, blockFactory, listener, transactionExecutorFactory, signatureCache, outdatedThreshold, outdatedTimeout, txQuotaChecker, gasPriceTracker, null);
    }
```